docs/table-usage.rst

```diff
@@ -0,0 +1,46 @@
+Table Admin
+===========
+
+After creating an :class:`~google.cloud.spanner_v1.database.Database`, you can
+interact with individual tables for that instance.
+
+
+List Tables
+-----------
+
+To iterate over all existing tables for an database, use its
+:meth:`~google.cloud.spanner_v1.database.Database.list_tables` method:
+
+.. code:: python
+
+    for table in database.list_tables():
+        # `table` is a `Table` object.
+
+This method yields :class:`~google.cloud.spanner_v1.table.Table` objects.
+
+
+Table Factory
+-------------
+
+A table object can be created by using the
+:class:`~google.cloud.spanner_v1.table.Table` constructor. Since table
+operations are executed via SQL, a
+:class:`~google.cloud.spanner_v1.database.Database` instance is required.
+
+.. code:: python
+
+    table = google.cloud.spanner_v1.table.Table(
```

Comment:
Please add a `table()` method on the `Database` object which creates the `Table` object to be consistent with the other classes e.g.
```
client = spanner.Client()
instance = client.instance(instance_id)
database = instance.database(database_id)
table = database.table(table_id)
```

Reply:
Done! Thanks for your patience while I was OOO.